README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# space-travelers-hub
+This is  a web application for a company that provides commercial and scientific space travel services. The application allows users to book rockets and join selected space missions.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Below is the screenshot of my work
+![space-travelers-hub](https://github.com/ndikumanaisaie/space-travelers-hub/blob/use-Redux-in-React/src/assets/images/shot.png)
 
-## Available Scripts
+## Built with
+- Javascript
+- React
+- SpaceX api
+- Redux
+- gitflow
 
-In the project directory, you can run:
+## Author
 
-### `npm start`
+👤 **Ndikumana Isaie**
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- GitHub: [ndikumanaisaie](https://github.com/ndikumanaisaie)
+- Twitter: [Ndikuma38670724](https://twitter.com/Ndikuma38670724)
+- LinkedIn: [Ndikumana Isaie](https://www.linkedin.com/in/ndikumanaisaie/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+👤 **Aminu Rabiu**
 
-### `npm test`
+- GitHub: [@arabiu033](https://github.com/arabiu033)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/larabiu033)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Link to URL on gitHub
+- [space-travelers-hub](https://github.com/arabiu033/space-travelers-hub.git)
 
-### `npm run build`
+## Link to URL in the browser
+- [space-travelers-hub](https://bookcollectionreact.netlify.app/)
```

Comment:
- [ ] Kindly provide the actual live demo of the project. Currently, you are using a different project link.

Reply:
Done.